src/Security/Permission.php

```diff
@@ -688,6 +690,7 @@ public static function get_declared_permissions_list()
      */
     public static function get_label_for_permission($perm)
     {
+        Deprecation::notice('4.4.0');
```

Comment:
Add a standardised message here and to the phpdoc

Reply:
Updated